appsdk/sdk_test.go

```diff
@@ -436,13 +426,13 @@ func TestInitializeClientsAll(t *testing.T) {
 }
 
 func TestInitializeClientsJustData(t *testing.T) {
-	clients := make(map[string]common.ClientInfo)
-	clients[common.CoreDataClientName] = common.ClientInfo{}
+	clientMap := make(map[string]common.ClientInfo)
```

Comment:
Don't use type in the variable name. Leave as clients

Reply:
`clients` collides with a package name and generates a linter warning.